src/gui/DatabaseTabWidget.cpp

```diff
@@ -607,6 +613,39 @@ void DatabaseTabWidget::updateTabNameFromDbWidgetSender()
 
     DatabaseWidget* dbWidget = static_cast<DatabaseWidget*>(sender());
     updateTabName(databaseFromDatabaseWidget(dbWidget));
+
+    Database* db = dbWidget->database();
+    Group *autoload = db->rootGroup()->findChildByName("AutoOpen");
+    if (autoload)
+    {
```

Comment:
Coding style: pointer operator belongs to type, opening brace on same line as if.